lib/extensions/print_pdf.py

```diff
@@ -274,6 +274,8 @@ def render_html(self, stitch_plan, overview_svg, selected_palette):
                 'client_detailedview': False,
                 'operator_overview': True,
                 'operator_detailedview': True,
+                'pattern_onlyview': False,
+                'pattern_onlyview': False,
```

Comment:
This looks like it got added twice. (Sorry for the drive-by comment)

Reply:
Thanks for catching that!